src/tasks/level6.ts

```diff
@@ -46,6 +47,10 @@ export const FriarQuest: Quest = {
     {
       name: "Neck",
       after: ["Start"],
+      prepare: () => {
+        tryForceNC()
```

Comment:
We probably want to add these on the other friar tasks too, no?

Reply:
I wasn't sure; we wanted to charge Latte, and forcing NC forecloses that opportunity.